confidant_client/__init__.py

```diff
@@ -87,6 +88,7 @@ def __init__(
             configuration from. First file found will be used. Default:
                 ['~/.confidant', '/etc/confidant/config']
             profile: profile to read config values from.
+            verify:  Whether we verify the server’s TLS certificate.
```

Comment:
It looks like the ' here is unicode, which is causing tests to fail.

Reply:
Thanks for spotting it, Looks like my copy paste from the official documentation. Ill fix and update PR.